horovod/ray/runner.py

```diff
@@ -0,0 +1,445 @@
+import ray
+
+from collections import defaultdict
+from dataclasses import dataclass
+import os
+from typing import Dict, Callable, Any, Optional, List
+import logging
+
+from horovod.runner.common.util import secret, timeout, hosts
+from horovod.runner.driver import driver_service
+from horovod.runner.http.http_server import RendezvousServer
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class MiniSettings:
+    """Mock object, only necessary for Tune.
```

Comment:
Is this docstring still correct?  Seems this correctly encapsulates the settings needed for Ray.